python/dask_cudf/dask_cudf/groupby.py

```diff
@@ -829,27 +835,27 @@ def _finalize_gb_agg(
         agg_list = aggs.get(col, [])
         agg_set = set(agg_list)
         if agg_set.intersection({"mean", "std", "var"}):
-            count_name = _make_name(col, "count", sep=sep)
-            sum_name = _make_name(col, "sum", sep=sep)
+            count_name = _make_name((col, "count"), sep=sep)
```

Comment:
If you followed my suggestion above on returning lists these calls would become something like...
```suggestion
            count_name, *_ = _make_name((col, "count"), sep=sep)
```